pandas/tests/series/test_npfuncs.py

```diff
@@ -0,0 +1,54 @@
+"""
+Tests for np.foo applied to Series, not necessarily ufuncs.
+"""
+
+import numpy as np
+import pytest
+
+from pandas import Series, Timestamp, date_range
+import pandas._testing as tm
+
+
+class TestAsArray:
+    @pytest.mark.parametrize("tz", [None, "US/Central"])
+    def test_asarray_object_dt64(self, tz):
+        ser = Series(date_range("2000", periods=2, tz=tz))
```

Comment:
hmm, these are not npfuncs, i think this would be better in a test_datetimes?

Reply:
its for `np.asarray`

current plan is to get rid of tests.(series|frame).test_datetimes

Reply:
i get it but these are pure conversion tests should be located with to_numpy() for example and it ufuncs themselves

Reply:
there is already a dedicated test_ufuncs.py.

The idea behind the collect-by-method approach is for it to be really obvious where a test belongs.  to_numpy gets tested in test_to_numpy, .values gets tested in test_values.

np.array is an odd duck because it isnt one of our methods, so this may not be the best long-term place for this (ive been looking at tests.base which has tests shared between Index/Series which this might fit into eventually).  how about we revisit this in a few weeks when (hopefully) we're closer to the end of this process

Reply:
hmm i would just move it now

Reply:
updated to move these to tests.base.test_conversion